From 5206571c6a28c595d622c6164aac72bd87d240c0 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Fri, 27 Jul 2018 14:48:08 +0800
Subject: [PATCH 4248/5242] MLK-19067: ARM64: dts: fix dp audio is not probed

commit  71f8ad9cccc375f7fb702a22650984db12b46e1b from
https://source.codeaurora.org/external/imx/linux-imx.git

With the commit c5592424ba05 ("MLK-18368-2: ASoC: fsl: support
hdmi rx in machine driver"), hdmi-out/hdmi-in need to specified
in dts.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts    |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
index 5647d8ff..666feaf 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
@@ -23,6 +23,7 @@
 		model = "imx-audio-dp";
 		audio-cpu = <&sai_hdmi_tx>;
 		protocol = <1>;
+		hdmi-out;
 	};
 };
 
-- 
1.7.9.5

